### PR TITLE
Update security policy and add bug report banner

### PR DESCRIPTION
**Pull request by qaisjp**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -5,6 +5,23 @@ labels: bug
...
+ |_ _|  \/  |  _ \ / _ \|  _ \_   _|/ \  | \ | |_   _|
+  | || |\/| | |_) | | | | |_) || | / _ \ |  \| | | |  
+  | || |  | |  __/| |_| |  _ < | |/ ___ \| |\  | | |  
+ |___|_|  |_|_|    \___/|_| \_\|_/_/   \_\_| \_| |_|     
+
+IF YOU ARE REPORTING A CHEAT OR A SECURITY VULNERABILITY... STOP!
+
+DO NOT MAKE A PUBLIC POST. GO BACK. CLICK THE 'VIEW POLICY' BUTTON.
+
+IF YOU ARE LOST AND CANNOT FIND A LINK, HERE IT IS:
```
Comment (buepas) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Looks good until here,
```suggestion
    IF YOU HAVE TROUBLE FINDING THE RIGHT LINK:
```

**Review by buepas — CHANGES_REQUESTED** [review]
Looks good so far,

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (buepas) [reply .github/ISSUE_TEMPLATE/bug_report.md]: because the other sounds a little too harsh

Comment (qaisjp) [reply .github/ISSUE_TEMPLATE/bug_report.md]: thanks, good call

**Review by buepas — APPROVED** [review]

**Review by sbx320 — APPROVED** [review]

**Comment by Pirulax** [comment]
Look good to me.

**Pull request merged**